src/lib/update-alternatives/model/alternative.rb

```diff
@@ -58,17 +58,17 @@ def self.load(name)
       alternative = parse_to_map(raw_data.slice(0..raw_data.find_index("\n")))
       choices = raw_data.slice(raw_data.find_index("\n") + 1..raw_data.length)
       new(
-        alternative["Name:"],
-        alternative["Status:"],
-        alternative["Value:"],
+        alternative["Name"],
+        alternative["Status"],
+        alternative["Value"],
         load_choices_from(choices)
       )
     end
 
     def self.parse_to_map(alternative_data)
       alternative = {}
       alternative_data.each do |line|
-        alternative[line.split.first] = line.split.last
+        alternative[line.split(/\W/).first] = line.split.last
```

Comment:
for me how can code looks like is e.g. 
```
# output line looks like "Name: vim"
key, value = line.split(":", 2)
alternative[key.strip] = value.strip
```

this way it is clear what it is done, what is expected input ( so you can compare it to reality if it changes ) and also it nicely handle if you have path with spaces like "/usr/bin/my editor" or "/usr/bin/my:editor"